interview-questions.md

```diff
@@ -7,60 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is the program stored in the Learn computer to store code changes made by a group of coders.
 
-  Researched answer:
+  Researched answer:  "Git is a software that tracks changes made to any set of files. Wikipedia.
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: True or False
```

Comment:
Well, yes Boolean values are true and false. But I am looking for what will get us a Boolean value. We know that relational, logical, and equality operators return Boolean values.